flavor_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "prodUrl": "https://api.trakt.tv",
+  "sandBoxUrl": "https://api-staging.trakt.tv",
+  "prodTitle": "Star Movie",
+  "sandBoxTitle": "Star Movie - SandBox"
+}
```

Comment:
split configs into 2 files - prod_config and sandbox_config. in this case u can use same key names, like base_url and app_title